### PR TITLE
add migaloo chain

### PR DESCRIPTION
**Pull request by 0xbobloblaw**

### CODE REVIEW

File: cosmos/migaloo.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		"coinDenom": "WHALE",
+		"coinMinimalDenom": "uwhale",
+		"coinDecimals": 6,
+		"coinGeckoId": "unknown"
+	},
+	"gasPriceStep": {
+		"low": 0.01,
+		"average": 0.025,
+		"high": 0.03
+	},
```
Comment (HeesungB) [inline cosmos/migaloo.json:50]: Could you remove `gasPriceStep`this field?

Comment (HeesungB) [reply cosmos/migaloo.json:50]: @0xbobloblaw @Caleboy4

Comment (0xbobloblaw) [reply cosmos/migaloo.json]: done @HeesungB

**Comment by HeesungB** [comment]
@0xbobloblaw @Caleboy4 Thank you for contribution!

**Pull request merged**